memory_core/memory_core_magma.py

```diff
@@ -67,27 +67,26 @@ def __init__(self,
                  banks=1,
                  input_iterator_support=6,  # Addr Controllers
                  output_iterator_support=6,
-                 input_config_width=16,
-                 output_config_width=16,
+                 config_width=16,
                  interconnect_input_ports=2,  # Connection to int
                  interconnect_output_ports=2,
                  mem_input_ports=1,
                  mem_output_ports=1,
-                 use_sram_stub=True,
+                 use_sim_sram=True,
                  sram_macro_info=SRAMMacroInfo("TS1N16FFCLLSBLVTC512X32M4S",
                                                wtsel_value=0, rtsel_value=1),
                  read_delay=1,  # Cycle delay in read (SRAM vs Register File)
                  rw_same_cycle=False,  # Does the memory allow r+w in same cycle?
                  agg_height=4,
-                 tb_sched_max=16,
                  config_data_width=32,
                  config_addr_width=8,
                  num_tiles=1,
                  fifo_mode=True,
                  add_clk_enable=True,
                  add_flush=True,
                  override_name=None,
-                 gen_addr=True):
+                 gen_addr=True,
+                 GF=True):
```

Comment:
Probably better to leave it as TSMC for now. I can change it to GF as part of my future PR

Reply:
Defaulted it to tsmc